### PR TITLE
LLM: add more example usages about alpaca qlora on different hardware

### PR DESCRIPTION
**Pull request by plusbang**
## Description

Add more example usages about alpaca qlora on different hardware

### CODE REVIEW

File: python/llm/example/GPU/QLoRA-FineTuning/alpaca-qlora/finetune_llama2_7b_pvc_1100_1_card.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# You could also specify `--base_model` to the local path of the huggingface model checkpoint folder and `--data_path` to the local path of the dataset JSON file
+python ./alpaca_qlora_finetuning.py \
+    --micro_batch_size 12 \
+    --batch_size 120 \
```
Comment (yangw1234) [inline python/llm/example/GPU/QLoRA-FineTuning/alpaca-qlora/finetune_llama2_7b_pvc_1100_1_card.sh:20]: Second thought, maybe we can just use `--micro_batch_size 8 --batch_size 128`.

File: python/llm/example/GPU/QLoRA-FineTuning/alpaca-qlora/finetune_llama2_7b_pvc_1100_4_card.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+export MASTER_ADDR=127.0.0.1
+export OMP_NUM_THREADS=28 # adjust this to 1/4 of total physical cores
+torchrun --standalone \
+         --nnodes=1 \
+         --nproc-per-node 4 \
+         ./alpaca_qlora_finetuning.py \
+         --base_model "meta-llama/Llama-2-7b-hf" \
+         --data_path "yahma/alpaca-cleaned" \
+         --output_dir "./bigdl-qlora-alpaca" \
+         --micro_batch_size 12 \
```
Comment (yangw1234) [inline python/llm/example/GPU/QLoRA-FineTuning/alpaca-qlora/finetune_llama2_7b_pvc_1100_4_card.sh:27]: Also `--micro_batch_size 8 --batch_size 128`?

File: python/llm/example/GPU/QLoRA-FineTuning/alpaca-qlora/finetune_llama2_7b_pvc_1100_1_card.sh (hunk 1 above)
Comment (plusbang) [reply python/llm/example/GPU/QLoRA-FineTuning/alpaca-qlora/finetune_llama2_7b_pvc_1100_1_card.sh]: > Second thought, maybe we can just use `--micro_batch_size 8 --batch_size 128`.

Have updated to make it consistent.

**Review by yangw1234 — APPROVED** [review]
LGTM

**Pull request merged**